### PR TITLE
Update copyrights on non-source files

### PR DESCRIPTION
**Pull request by marabooy**
### Issues

Update copyrights on non source files
### Description

*Briefly describe the changes of this pull request.*

### Checklist (Uncheck if it is not completed)

- [x] *Test cases added*
- [x] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

**Review by xuzhg — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
  Build  | Status
 --------|---------
-WebApi Rolling | <img src="https://identitydivision.visualstudio.com/OData/_apis/build/status/WebApi/WebApi-master-pipeline-Rolling"/>
```
Comment (xuzhg) [inline README.md:5]: If the existing rolling is still running, can we keep the badge

File: README.md [hunk 2]
```diff
@@ -118,4 +119,10 @@ Please refer to the [How to debug](https://docs.microsoft.com/en-us/odata/webapi
...
 ### Code of Conduct
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+This project has adopted the [.NET Foundation Contributor Covenant Code of Conduct](https://dotnetfoundation.org/about/code-of-conduct). For more information see the [Code of Conduct FAQ](https://dotnetfoundation.org/about/faq).
+
+### .NET Foundation
+
+This project is supported by the [.NET Foundation](https://dotnetfoundation.org).
+
+OData Connected Service is a Copyright &copy; .NET Foundation and other contributors. It is licensed under [MIT License](https://github.com/OData/WebApi/blob/master/License.txt)
```
Comment (gathogojr) [inline README.md:128]: Replace "OData Connected Service"

Comment (gathogojr) [reply README.md:128]: Missing "of" between &copy; and .NET?

**Review by gathogojr — APPROVED** [review]
Left a couple of comments otherwise LGTM

**Pull request merged**